client/app/dbaas/logs/detail/streams/alerts/add/alerts-add.html

```diff
@@ -1,5 +1,195 @@
+                            data-ng-maxlength="50"
+                            maxlength="50"
+                            required />
+                </div>
+                <div class="oui-field-helper" data-translate="add_alert_field_field_description"></div>
+            </div>
+        </div>
+
+        <div data-ng-if="ctrl.alertType === 'FIELD_CONTENT_VALUE'" class="oui-field" data-ng-class="{'oui-field_error': ctrl.form.$submitted && ctrl.form.val.$invalid}">
+                <label for="val" class="oui-field__label oui-label" data-translate="add_alert_value_field"></label>
+                <div class="oui-field__content">
+                    <div class="oui-field-control oui-field-control_4">
+                        <input type="text" id="val" name="val"
+                                class="oui-input" 
+                                data-ng-model="ctrl.alert.value"
+                                data-ng-class="{'oui-input_error': ctrl.form.$submitted && ctrl.form.field.$invalid}"
+                                data-ng-minlength="3"
+                                data-ng-maxlength="50"
+                                maxlength="50"
+                                required />
+                    </div>
+                    <div class="oui-field-helper" data-translate="add_alert_value_field_description"></div>
+                </div>
+            </div>
+
+        <div data-ng-if="ctrl.alertType !== 'FIELD_CONTENT_VALUE'">
+            <div class="oui-field" data-ng-class="{'oui-field_error': ctrl.form.$submitted && ctrl.form.time_period.$invalid}">
+                <label for="time_period" class="oui-field__label oui-label" data-translate="add_alert_time_period_field"></label>
+                <div class="oui-field__content">
+                    <oui-numeric
+                        id="time_period"
+                        name="time_period"
+                        model="ctrl.alert.time"
+                        min="1"
+                        max="3600"
+                        required>
+                    </oui-numeric>
+                    <div class="oui-field-helper" data-translate="add_alert_time_period_field_description"></div>
+                </div>
+            </div>
+
+            <div class="oui-field">
+                <label for="threshold_type" class="oui-label" data-translate="add_alert_threshold_type_field"></label>
+                <div class="oui-field__content">
+                    <div class="oui-field-control oui-field-control_4">
+                        <div class="oui-select">
+                            <select class="oui-select__input"
+                                name="threshold_type"
+                                id="threshold_type"
+                                data-ng-model="ctrl.alert.thresholdType">
+                                <option data-ng-if="ctrl.alertType !== 'FIELD_VALUE'" value="MORE" data-translate="add_alert_threshold_type_more"></option>
```

Comment:
This list of options should be in the controller.
If, one day, the Enum comes from the API, you will be able to change it more easily.

Reply:
This is done. The controller returns the options from the constants